### DISCUSSION TITLE
FitFusion

### DISCUSSION THREAD

**Discussion by giscus** [JirenXmoksh/FEE-II-PRJ#13] [General]
# FitFusion

http://127.0.0.1:5500/UI/FEE%20-%20II/PRJEXC/source/index.htm

**Comment by JirenXmoksh** [comment]
This is me! JirenXmoksh, hello everyone.

**Reply by Himanshu1237** [reply]
Why isn't there a downvote option ? 😀

**Reply by JirenXmoksh** [reply]
Because there's nothing called down apart from the exercise movements at FITFUSION!

**Comment by Himanshu1237** [comment]
Have a Good Day!
